.clang-format

```diff
@@ -0,0 +1,104 @@
+# OSARA: Open Source Accessibility for the REAPER Application
+# .clang-format
+# Author: James Teh <jamie@jantrid.net>
+# Copyright 2023 Leonard de Ruijter
+# License: GNU General Public License version 2.0
+
+# Style based on Google's C++ style guide.
+BasedOnStyle: Google
+# Offset for access modifiers (public, protected, private).
+AccessModifierOffset: 0
+# Align code after an open bracket with block indentation.
+AlignAfterOpenBracket: BlockIndent
+# Do not align consecutive assignments.
+AlignConsecutiveAssignments: None
+# Do not align consecutive bit fields.
+AlignConsecutiveBitFields: None
+# Do not align consecutive declarations.
+AlignConsecutiveDeclarations: None
+# Do not align consecutive macros.
+AlignConsecutiveMacros: None
+# Do not align escaped newlines.
+AlignEscapedNewlines: DontAlign
+# Do not align operands in expressions.
+AlignOperands: DontAlign
+# Do not align trailing comments.
+AlignTrailingComments: Never
+# Do not allow short blocks (e.g., if statements without braces) on a single line.
+AllowShortBlocksOnASingleLine: Never
+# Do not allow short case labels in switch statements on a single line.
+AllowShortCaseLabelsOnASingleLine: false
+# Do not allow short enums on a single line.
+AllowShortEnumsOnASingleLine: false
+# Only allow empty functions on a single line.
+AllowShortFunctionsOnASingleLine: Empty
+# Do not allow short if statements on a single line.
+AllowShortIfStatementsOnASingleLine: Never
+# Do not allow short loops on a single line.
+AllowShortLoopsOnASingleLine: false
+# Do not pack function call arguments together.
+# A function call’s arguments will either be all on the same line or will have one line each.
+BinPackArguments: false
+# Do not pack function parameters together.
+# A function declaration’s or function definition’s parameters will either all be on the same line
+# or will have one line each.
+BinPackParameters: false
+# Add a space after the colon in bit fields.
+BitFieldColonSpacing: After
+# Break before binary operators that are not assignments.
+# E.g. break before "+", but not before "+=".
+BreakBeforeBinaryOperators: NonAssignment
+# Attach opening braces to the same line as the statement or declaration it belongs to.
+BreakBeforeBraces: Attach
+# Break string literals if they are too long.
+BreakStringLiterals: true
+# Maximum column width before code will be wrapped or formatted.
+ColumnLimit: 110
```

Comment:
I'm weird about maximum line widths in general, but is there a reason you chose 110? There seems to be a fair amount of standardisation around 80 and I'm trying to conform to this in newer code, even though I didn't in older code. I don't really think 80 should be significant these days, but in the absence of a good argument, I'm inclined to lean towards the most common convention.

Reply:
I choose 110 because it is used in NVDA 😳
How about leaving it out? Then we'll stick to Google's default, whatever it is. Probably 79 or something.